src/components/cards/ProjectCard.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import CardContainer from './CardContainer';
+import CardImage from './CardImage';
+import CardTitle from './CardTitle';
+import CardAvatar from './CardAvatar';
+
+const ProjectCard = ({
+  key,
+  projectUrl,
+  projectImageUrl,
+  avatarUrl,
+  ownerProfileUrl,
+  title,
+  type
+}) => {
+  return (
+    <CardContainer className="card-project" key={key}>
+      <Link to={projectUrl}>
```

Comment:
We'll have to define the routes for the different pages on the frontend. You can update this to 
```suggestion
      <Link to={`projects/${key}`}>
```
since key contains the project id, you should rename the variable key to id for readability

Reply:
I think it would look messy to have a project ID as a uuid in the URL. Perhaps we change it to another form, like auto incrementing (1, 2, 3, 4 ... n)?